lib/graphql/enum_type.rb

```diff
@@ -136,7 +138,7 @@ def initialize(name:, description:, deprecation_reason:, value:)
         @name = name
         @description = description
         @deprecation_reason = deprecation_reason
-        @value = value
+        @value = value || name
```

Comment:
Can you help me understand why this was needed? I couldn't quite tell just by looking! (Usually this is covered by `GraphQL::Define::AssignEnumValue`)